admin/taxonomy/class-taxonomy.php

```diff
@@ -172,17 +173,35 @@ private function get_taxonomy() {
 	}
 
 	/**
-	 * Test whether we are on a public taxonomy - no metabox actions needed if we are not
-	 * Unfortunately we have to hook most everything in before the point where all taxonomies are registered and
-	 * we know which taxonomy is being requested, so we need to use this check in nearly every hooked in function.
 	 *
-	 * @since 1.5.0
+	 * @return array
 	 */
-	private function tax_is_public() {
-		// Don't make static as taxonomies may still be added during the run.
-		$taxonomies = get_taxonomies( array( 'public' => true ), 'names' );
+	public function localize_term_scraper_script() {
+		$file = plugin_dir_path( WPSEO_FILE ) . 'languages/wordpress-seo-' . get_locale() . '.json';
```

Comment:
See my comment above + this is the second time we do this, maybe we should abstract this.